src/main/java/net/logstash/logback/stacktrace/StackElementFilter.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.logstash.logback.stacktrace;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Component in charge of accepting or rejecting {@link StackTraceElement elements} when computing a stack trace hash
+ */
+public abstract class StackElementFilter {
```

Comment:
Good suggestion.  Yeah, probably too late to change `match()` to `find()` now.  Will update docs